plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -580,10 +580,7 @@ class Yarn2(
         val vcsFromPackage = parseNpmVcsInfo(packageJson)
         val homepage = packageJson.homepage.orEmpty()
         val author = parseNpmAuthors(packageJson.author)
-
-        var downloadUrl = packageJson.dist?.tarball.orEmpty()
-
-        downloadUrl = fixNpmDownloadUrl(downloadUrl)
+        val downloadUrl = packageJson.dist?.tarball.orEmpty().fixNpmDownloadUrl()
```

Comment:
> makes the use of a `var` unnecessary.

Couldn't the same be achieved by using

    val downloadUrl = fixNpmDownloadUrl(packageJson.dist?.tarball.orEmpty())

here?